<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoWidget</class>
 <widget class="QWidget" name="VideoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Config</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="5">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>User:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="framesSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Width: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="widthSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>640</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Frames: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Height: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSpinBox" name="heightSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>480</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="portSpinBox">
        <property name="minimum">
         <number>1000</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>8080</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="11">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Camera 2</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Device:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="video1DeviceEdit">
           <property name="placeholderText">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="video1UpsideDownCheckBox">
           <property name="text">
            <string>Upside Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="11">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Camera 1</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Device: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="video0DeviceEdit">
           <property name="text">
            <string>0</string>
           </property>
           <property name="placeholderText">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="video0UpsideDownCheckBox">
           <property name="text">
            <string>Upside Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" colspan="11">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Camera 3</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Device: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="video2DeviceEdit">
           <property name="placeholderText">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="video2UpsideDownCheckBox">
           <property name="text">
            <string>Upside Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0" colspan="11">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Camera 4</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Device: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="video3DeviceEdit">
           <property name="placeholderText">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="video3UpsideDownCheckBox">
           <property name="text">
            <string>Upside Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QLineEdit" name="passwordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" colspan="2">
       <widget class="QLineEdit" name="userEdit"/>
      </item>
      <item row="2" column="5" colspan="6">
       <widget class="QCheckBox" name="findPersonsCheckBox">
        <property name="text">
         <string>find Persons in Video and send Movement</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="urlLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
